src/main/java/command/AddCommand.java

```diff
@@ -0,0 +1,115 @@
+package command;
+
+import storage.Storage;
+import task.TaskList;
+import ui.Ui;
+import exception.DukeException;
+import task.Task;
+import task.Todo;
+import task.Deadline;
+import task.Event;
+
+/**
+ * Represents an Add Command that inherits from
+ * the abstract class Command.
+ */
+
+public class AddCommand extends Command {
+    protected String commandLine;
```

Comment:
I didn't like the variable name for this, perhaps userInput is more intuitive? 🤔